authority/provisioner/provisioner.go

```diff
@@ -29,6 +30,7 @@ type Interface interface {
 	AuthorizeSSHRevoke(ctx context.Context, token string) error
 	AuthorizeSSHRenew(ctx context.Context, token string) (*ssh.Certificate, error)
 	AuthorizeSSHRekey(ctx context.Context, token string) (*ssh.Certificate, []SignOption, error)
+	DefaultTLSCertDuration() time.Duration
```

Comment:
If you want to avoid adding this, you can make add a new interface acme, `acme.Provisioner`, that either extends for `provisioner.Interface` and adds that method, or perhaps, just define the methods required.